spec/passkey-server.yaml

```diff
@@ -430,10 +509,6 @@ components:
                 type:
                   - integer
                   - 'null'
-              additional:
-                type:
-                  - object
-                  - 'null'
```

Comment:
This property still exists and was not removed.

Reply:
Fixed